src/components/clock/Clock.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+type Props = {
+  clockName: string;
+};
+
+type State = {
+  date: Date;
+};
+
+export class Clock extends React.Component<Props, State> {
+  state: Readonly<State> = {
+    date: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount(): void {
+    this.timerId = window.setInterval(() => {
+      this.setState({
+        date: new Date(),
+      });
+
+      // eslint-disable-next-line no-console
+      console.info(this.state.date.toUTCString().slice(-12, -4));
```

Comment:
will be better to create separate function for getting date in needed format, because you use the same functionality twice in your code